src/CFNetwork/MessageHandler.cs

```diff
@@ -27,6 +27,8 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
+#nullable enable
```

Comment:
I would say that we should just rollback all the changes in this file since it is deprecated and since enabling nullability here would need us to look at some behavior changes 

```csharp
[Obsolete ("Use 'System.Net.Http.CFNetworkHandler' or the more recent 'Foundation.NSUrlSessionHandler' instead.")]
	public class MessageHandler : HttpClientHandler {
```